pyproject.toml

```diff
@@ -434,16 +434,19 @@ disable = [
   "bad-inline-option",
   "deprecated-pragma",
   "file-ignored",
+  "import-error", # pylint gets confused by tests for our examples
   "import-outside-toplevel", # Allow lazy imports inside of methods
   "locally-disabled",
   "missing-class-docstring",
   "missing-function-docstring",
   "missing-module-docstring",
+  "no-member", # pylint isn't aware of some of our attrs
```

Comment:
Lets make an issue to address this with changing the check and build stage order in CI